### ISSUE TITLE
Some Dataframe commands are interpreted as external commands in some circumstances

### ISSUE THREAD

**Issue by PhilVince96** [nushell/nushell#8288]
### Describe the bug

As I go through the [Nushell Book](https://nushell.sh/book/dataframes.html#series-as-indices) I got stuck in the dataframes section "series" because some dfr commands are not recognised although they are installed.
dfr value-counts, dfr arg-unique, dfr arg-sort are some examples. When you go to help dfr value-counts and try the example, the command does work properly however. Same thing for the other mentioned commands.

### How to reproduce

1. Load the test_small.csv into a dataframe $df
2. $df.first | dfr value-counts
3. I get an error message

### Expected behavior

A dataframe with all the same values counted.

### Screenshots

<img width="547" alt="Screenshot 2023-03-02 at 11 43 07" src="https://user-images.githubusercontent.com/29581161/222406221-66a016bc-973e-4a04-8649-9e8d24d0302b.png">

### Configuration

| key                | value                                           |
| ------------------ | ----------------------------------------------- |
| version            | 0.76.0                                          |
| branch             |                                                 |
| commit_hash        |                                                 |
| build_os           | macos-aarch64                                   |
| build_target       | aarch64-apple-darwin                            |
| rust_version       | rustc 1.67.1 (d5a82bbd2 2023-02-07)             |
| rust_channel       | stable-aarch64-apple-darwin                     |
| cargo_version      | cargo 1.67.1 (8ecd4f20a 2023-01-10)             |
| pkg_version        | 0.76.0                                          |
| build_time         | 2023-03-02 11:21:45 +01:00                      |
| build_rust_channel | release                                         |
| features           | database, dataframe, default, trash, which, zip |
| installed_plugins  | hist, plot, xyplot                              |

**Comment by fdncred** [comment]
I agree that something weird is going on. I can only get it to work if coming from `dfr into-df` like this
```
ls | dfr into-df | get size | dfr into-df | dfr value-counts
```
This does not work but it seems like it should
```
ls | dfr into-df | get size | dfr value-counts
```

**Comment by sholderbach** [comment]
This was likely down to the input type based command overloading associated with the old `dfr` commands. This system has been removed and the `dfr` commands moved into the `polars` plugin.
